cmd/athenz-sia/main.go

```diff
@@ -51,6 +51,7 @@ func parseFlags(program string, args []string) (*identity.IdentityConfig, error)
 		podIP           = envOrDefault("POD_IP", "")
 		podUID          = envOrDefault("POD_UID", "")
 		saTokenFile     = envOrDefault("SA_TOKEN_FILE", "/var/run/secrets/kubernetes.io/bound-serviceaccount/token")
+		serverCACert    = envOrDefault("SERER_CA_CERT", "")
```

Comment:
Minor: Can we fix the typo here? `SERER_CA_CERT` -> `SERVER_CA_CERT`

Reply:
Fix: 50f7af6